packages/amplify-util-node-template-provider/amplify-plugin.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "function-template-provider",
+  "type": "util",
+  "commands": [],
+  "eventHandlers": [],
+  "implements": ["FunctionTemplateProvider"]
```

Comment:
it would be nice to not to leak implenentation details into json files like a class/interface name, but we already discussed it and it will be different here as well.

Reply:
I've update this based on our conversation offline.